### PR TITLE
Specifying why the -t is required on each fly command

### PR DESCRIPTION
**Pull request by Pivotal-Jayson-Hairston**

### CODE REVIEW

File: docs/using-concourse/fly-cli.any [hunk 1]
```diff
@@ -76,6 +76,10 @@ flag. Once you've learned what the commands do, you may want to consult
   example} for short) to run a command against the saved target \code{example}.
   For eample, \code{fly -t example builds} will list the last few builds on the
   \code{example} Concourse instance.
+  
+  The \code{fly} target option is intentionally stateless and must be explicitly added
+  to each command. This reduces the risk of acidentally running a command against the
```
Comment (vito) [inline docs/using-concourse/fly-cli.any]: typo (acidentally -> accidentally)

File: docs/using-concourse/fly-cli.any [hunk 2]
```diff
@@ -76,6 +76,10 @@ flag. Once you've learned what the commands do, you may want to consult
   example} for short) to run a command against the saved target \code{example}.
   For eample, \code{fly -t example builds} will list the last few builds on the
   \code{example} Concourse instance.
+  
+  The \code{fly} target option is intentionally stateless and must be explicitly added
```
Comment (vito) [inline docs/using-concourse/fly-cli.any]: hmmm this wording feels a bit confusing, maybe something like "The \code{-t} flag is intentionally stateless [...]"

Comment (Pivotal-Jayson-Hairston) [inline docs/using-concourse/fly-cli.any]: Good point, I have updated the wording now.

File: docs/using-concourse/fly-cli.any (hunk 1 above)
Comment (Pivotal-Jayson-Hairston) [inline docs/using-concourse/fly-cli.any]: Wow, somehow missed this. Fixed it now! Thanks!

**Comment by Pivotal-Jayson-Hairston** [comment]
@vito Please review again now as changes have been made.

**Comment by vito** [comment]
looks good! thanks.

**Pull request merged**